app/psc/app.toml

```diff
@@ -0,0 +1,354 @@
+size = 131072
+read = true
+write = true
+execute = false  # let's assume XN until proven otherwise
+
+[tasks.jefe]
+path = "../../task/jefe"
+name = "task-jefe"
+priority = 0
+requires = {flash = 8192, ram = 2048}
+start = true
+features = ["itm"]
+stacksize = 1536
+
+[tasks.sys]
+path = "../../drv/stm32xx-sys"
+name = "drv-stm32xx-sys"
+features = ["h753"]
+priority = 1
+requires = {flash = 8192, ram = 1024}
+uses = ["rcc", "gpios1", "gpios2", "gpios3"]
+start = true
+
+[tasks.spi4_driver]
+path = "../../drv/stm32h7-spi-server"
+name = "drv-stm32h7-spi-server"
+priority = 2
+requires = {flash = 16384, ram = 2048}
+features = ["spi4", "h753"]
+uses = ["spi4"]
+start = true
+interrupts = {"spi4.irq" = 1}
+stacksize = 872
+task-slots = ["sys"]
+
+[tasks.spi4_driver.config.spi]
+global_config = "spi4"
+
+[tasks.spi2_driver]
+path = "../../drv/stm32h7-spi-server"
+name = "drv-stm32h7-spi-server"
+priority = 2
+requires = {flash = 16384, ram = 2048}
+features = ["spi2", "h753"]
+uses = ["spi2"]
+start = true
+interrupts = {36 = 1}
+stacksize = 872
+task-slots = ["sys"]
+
+[tasks.spi2_driver.config.spi]
+global_config = "spi2"
+
+[tasks.i2c_driver]
+path = "../../drv/stm32h7-i2c-server"
+name = "drv-stm32h7-i2c-server"
+features = ["h753", "itm"]
+priority = 2
+requires = {flash = 16384, ram = 2048}
+uses = ["i2c2", "i2c3", "i2c4"]
+start = true
+task-slots = ["sys"]
+
+[tasks.i2c_driver.interrupts]
+33 = 0b0000_0010        # I2C2 event
+34 = 0b0000_0010        # I2C2 error
+72 = 0b0000_0100        # I2C3 event
+73 = 0b0000_0100        # I2C3 error
+95 = 0b0000_1000        # I2C4 event
+96 = 0b0000_1000        # I2C4 error
+
+[tasks.hiffy]
+path = "../../task/hiffy"
+name = "task-hiffy"
+features = ["h753", "stm32h7", "itm", "i2c", "gpio", "spi"]
+priority = 3
+requires = {flash = 32768, ram = 16384 }
+stacksize = 1024
+start = true
+task-slots = ["sys", "hf", "i2c_driver"]
+
+[tasks.idle]
+path = "../../task/idle"
+name = "task-idle"
+priority = 5
+requires = {flash = 128, ram = 256}
+stacksize = 256
+start = true
+
+[peripherals.rcc]
+address = 0x58024400
+size = 1024
+
+[peripherals.gpios1]
+address = 0x58020000
+size = 0x2000
+
+[peripherals.gpios2]
+address = 0x58022000
+size = 0x0800
+
+[peripherals.gpios3]
+address = 0x58022800
+size = 0x0400
+
+[peripherals.spi2]
+address = 0x40003800
+size = 1024
+
+[peripherals.spi4]
+address = 0x40013400
+size = 1024
+interrupts = { irq = 84 }
+
+[peripherals.usart3]
+address = 0x40004800
+size = 1024
+
+[peripherals.i2c1]
+address = 0x40005400
+size = 1024
+
+[peripherals.i2c2]
+address = 0x40005800
+size = 1024
+
+[peripherals.i2c3]
+address = 0x40005c00
+size = 1024
+
+[peripherals.i2c4]
+address = 0x58001c00
+size = 1024
+
+[peripherals.quadspi]
+address = 0x52005000
+size = 4096
+
+[config]
+
+#
+# I2C2: Local bus
+#
+[[config.i2c.controllers]]
+controller = 2
+
+#
+# SP_I2C_LOCAL_SDA
+# SP_I2C_LOCAL_SCL
+#
+[config.i2c.controllers.ports.F]
+name = "local"
+description = "M.2 bus"
```

Comment:
I don't think there's an M.2 bus here.

Reply:
Ha -- yes, thank you! Fixed the description.